content/saas/opsworks_migration.md

```diff
@@ -0,0 +1,139 @@
++++
+title = "Migrate from AWS OpsWorks to Chef SaaS"
+draft = false
+[menu]
+  [menu.saas]
+    title = "OpsWorks Migration"
+    identifier = "chef_infra/OpsWorks Migration"
+    parent = "chef_saas"
+    weight = 30
++++
+
+This guide will walk you through the migration scenarios from AWS OPSWorks to Chef SaaS.
+
+{{< note >}} Assuming that you have upgraded to Automate 2.0 {{< /note >}}
+
+If you have already upgraded to Chef Automate 2.0, no further installation, upgrade, or maintenance is required.
+
+## Prerequisites
+
+* Running Automate 2.0 in AWS OpsWorks.
+
+* A Chef SaaS Environment. Refer to the [Getting Started](/get_started/) with the Chef SaaS page.
+
+* An S3 bucket was provided by Progress Chef.
+
+## High-level steps to migrate from AWS OpsWorks to Chef SaaS
+
+The advantages of migrating from AWS OpsWorks to Chef SaaS are:
+
+* Customer reviews and reduced current data as needed.
+
+* Creates Data Backup.
+
+* Chef creates SaaS account.
+
+* Chef restores backup.
+
+* Register nodes with Chef SaaS.
+
+## Prepare DNS for cutover
+
+As part of the migration process, we deploy a Blue/Green method that links to your existing public DNS. Refer to an example shown in the code snippet below:
```

Comment:
blue/green isn't capitalized. This sentence doesn't really explain anything. What are they supposed to do with this code example?